### PR TITLE
Additional nodes

### PR DESCRIPTION
**Pull request by lelit**
As requested in issue #6, these are rebased, simplified and tested nodes I implemented in my peace branch.

I'm not done yet, as I have four more nodes: `RangeFunction`, `AArrayExpr`, `MultiAssignRef` and `AIndices`.

Maybe you can quickly browse the commits though, spotting style or whatever issues.

### CODE REVIEW

**Comment by lelit** [comment]
Thank you Aldo, I should have addressed all your observations.

I added remaining nodes, so I'm done with this: of course, feel free to request further changes.

File: psqlparse/nodes/primnodes.py [hunk 1]
```diff
@@ -112,4 +112,35 @@ def tables(self):
...
+
+    def __init__(self, obj):
+        self.expr = build_from_item(obj, 'expr')
+        self.result = build_from_item(obj, 'result')
+        self.location = obj.get('location')
+
+
+class NullTest(Node):
+
+    def __init__(self, obj):
```
Comment (alculquicondor) [inline psqlparse/nodes/primnodes.py]: Did you get to check `xpr` property?

Comment (lelit) [inline psqlparse/nodes/primnodes.py]: No, sorry.

Comment (lelit) [inline psqlparse/nodes/primnodes.py]: FWIW, libpg_query [does not seem](https://github.com/lfittl/libpg_query/blob/9.5-latest/src/pg_query_json_defs.c#L455) to output it, even.

Comment (alculquicondor) [inline psqlparse/nodes/primnodes.py]: sounds good

**Pull request merged**